ui/media/js/image-modifiers.js

```diff
@@ -234,6 +228,22 @@ function refreshTagsList() {
     editorModifierTagsList.appendChild(brk)
 }
 
+function toggleCardState(modifierName, makeActive) {
+    document.querySelector('#editor-modifiers').querySelectorAll('.modifier-card').forEach(card => {
```

Comment:
Instead of looping over every item, even after a card has been found, how about:

```
let cards = document.querySelector('#editor-modifiers').querySelectorAll('.modifier-card').filter(card => card.querySelector('.modifier-card-label').innerText === modifierName)
cards.forEach(card => {
  if (makeActive) ...
}

Reply:
Test comment